var/spack/repos/builtin/packages/easybuild/package.py

```diff
@@ -12,10 +12,16 @@ class Easybuild(PythonPackage):
     """
 
     homepage = 'http://hpcugent.github.io/easybuild/'
-    url      = 'https://pypi.io/packages/source/e/easybuild/easybuild-3.1.2.tar.gz'
+    url      = 'https://pypi.io/packages/source/e/easybuild/easybuild-4.0.0.tar.gz'
+    maintainers = ['boegel']
```

Comment:
What does being a maintainer imply exactly?

Is this about maintaining the support in Spack for installing EasyBuild?

Reply:
It basically means 2 things:

1. If someone has a problem trying to install `easybuild` with Spack, we'll ping you
2. If someone wants to modify the `easybuild` package in Spack, we'll ping you

It's pretty low commitment, as I don't think anyone else has ever complained about problems with `easybuild` before.

Reply:
OK, thanks for clarifying.

Also, of course never complains about problems with EasyBuild... ;)